src/builtin/tables.scrbl

```diff
@@ -29,12 +29,32 @@
 
 @docmodule["tables" #:noimport #t #:friendly-title "Tables"]{
 
-  Tables consist of a sequence of zero or more @italic{rows}, which each
-  contain an equal amount of entries in named @italic{columns}.
-
+There are many examples of tables in computing, with
+spreadsheets being the most obvious.
+                                                             
+A @pyret{Table} is made up of @bold{rows} and @bold{columns}. Each
+column has a name.  Each row has an equal number of columns,
+in the same order.  Each column may be assigned a type via an
+annotation, and each entry in that column will be checked against
+the annotation.
```

Comment:
Some reordering, for less bouncing back and forth between columns and rows:

A table is made up of rows and columns.  All rows have the same number of columns, in the same order.  Each column has a name. Each column may also be assigned a type via an annotation; if so, all entries in a column will then be checked against the annotation.